### ISSUE TITLE
a link to my first-year exercises (not very polished)

### ISSUE THREAD

**Issue by gasche** [ocaml-sf/learn-ocaml-corpus#23]
I recently finished the first iteration of my first-year, first-semester Learn-OCaml course.
I just uploaded the exercises that I wrote for the occasion to <https://github.com/gasche/paris8-corpus-2019>.
The exercises are in French, although I would be happy to translate them if relevant/useful to someone.

The graders are not very good in several places, in particular as I did not know at the time how to test anything else than structural equality with the solution. (In particular, for exercises specified to accept results in arbitrary order, my graders force the same order as my solution.) I would like to revisit this, but I am not sure of when to do it.

**Comment by yurug** [comment]
@gasche Can I integrate these exercises in the corpus in a directory named `paris8`?

**Comment by gasche** [comment]
Yes of course. I don't think they are very high-quality, but they may be useful to people that need more very-easy exercises (than what is provided by the MOOC, for example).

Note: while I have a communication channel open, can I mention https://github.com/ocaml-sf/learn-ocaml/issues/363 to you ? This year I wanted to direct a more advanced student to an instance with François' exercises, and I reached for hackojo but it is unavailable. (It's also not so nice that it is linked on the main Learn-OCaml website as a demo instance, and dead.) I guess I/we should learn how to do static deployments that do work, and then release François' exercises as static webpages.

**Comment by yurug** [comment]
Thanks!

The static public instance is up again!
